ts/store/reducers/__mock__/backendStatus.ts

```diff
@@ -269,6 +269,13 @@ export const baseRawBackendStatus: BackendStatus = {
       dataMatrixPosteEnabled: false
     },
     fci: {
+      enabled: false,
+      min_app_version: {
+        android: "0.0.0.0",
+        ios: "0.0.0.0"
+      }
+    },
+    lollipop: {
```

Comment:
Is lollipop needed?

Reply:
@francescopersico lollipop ff was introduced here https://github.com/pagopa/io-services-metadata/commit/151bbf771356ecf9bd36df2045f5ba4e9c0a382d so is in the last version (1.0.22) of content-definitions https://github.com/pagopa/io-services-metadata/releases/tag/1.0.22